app-devel/cscope/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=cscope
+PKGSEC=devel
+PKGDEP="ncurses glibc"
+PKGDES="Source code browsing tool based on interactive text screen"
+
+ABSHADOW=0
```

Comment:
Please add a comment as to why you are doing this

Reply:
The Makefile generated by configure installs the binary to the system directory by default.
Do I need to add this sentence to the define file? By the way, I have grep the define files of other projects and found no similar comments.

Reply:
> The Makefile generated by configure installs the binary to the system directory by default. Do I need to add this sentence to the define file? By the way, I have grep the define files of other projects and found no similar comments.

> By the way, I have grep the define files of other projects and found no similar comments.

A lot of this is due to the history of irregularly introduced packages, and this is something that we don't have time to deal with yet

Reply:
For some reasons, I just prepared to add commet in the patch, but I noticed that the patch has been merged, I should have prepared it earlier, I am very sorry.